upup/pkg/fi/cloudup/awstasks/elastic_ip.go

```diff
@@ -304,7 +304,6 @@ type cloudformationElasticIP struct {
 func (_ *ElasticIP) RenderCloudformation(t *cloudformation.CloudformationTarget, a, e, changes *ElasticIP) error {
 	tf := &cloudformationElasticIP{
 		Domain: aws.String("vpc"),
-		Tags:   buildCloudformationTags(e.Tags),
```

Comment:
Hmm that's disappointing  - does cloudformation not support tagging ElasticIPs?  We should probably warn if we're dropping tags here.  Or can we do this with a different cloudformation task?

Reply:
this still fails. Can we get this accepted?

This is output when run: Encountered unsupported property Tags